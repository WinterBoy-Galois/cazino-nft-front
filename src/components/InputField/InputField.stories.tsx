import React from 'react';
import { action } from '@storybook/addon-actions';

import InputField from '.';

export default {
  title: 'Components/InputField',
  component: InputField,
};

const data = {
  label: 'Username',
  value: 'sergioalvarez',
  onChangeValue: action('onChangeValue'),
};

export const Default = () => <InputField {...data} />;
