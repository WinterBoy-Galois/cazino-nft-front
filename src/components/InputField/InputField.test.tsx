import React from 'react';
import { render } from '@testing-library/react';

import InputField from '.';

test('renders InputField', () => {
  const { container } = render(<InputField />);
  expect(container).toMatchSnapshot();
});
