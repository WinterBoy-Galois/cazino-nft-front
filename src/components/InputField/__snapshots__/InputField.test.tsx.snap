// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders InputField 1`] = `
<div>
  <div
    class="inputField__container"
  >
    <div
      class="inputField__wrapper"
    >
      <label
        class="inputFieldLabel"
      />
      <input
        autocomplete="off"
        class="inputField"
        type="text"
        value=""
      />
    </div>
  </div>
</div>
`;
